task/export.go

```diff
@@ -98,11 +98,16 @@ func uploadFile(tctx *TaskContext, asset *api.Asset, content io.Reader) (*data.E
 			ipfs = clients.NewPinataClientAPIKey(p.APIKey, p.APISecret, extMetadata)
 		}
 	}
+
 	videoCID, metadata, err := ipfs.PinContent(tctx, "asset-"+asset.PlaybackID, contentType, content)
 	if err != nil {
 		return nil, err
 	}
 	template, nftMetadata := params.IPFS.NFTMetadataTemplate, params.IPFS.NFTMetadata
+	if template == api.NFTMetadataTemplatePlayer && asset.PlaybackURL == "" {
+		return nil, fmt.Errorf("cannot create player NFT for asset without playback URL")
```

Comment:
So does this disable the `player` template on create?

Reply:
No! For the `upload` task which is what we do on creation, we don't have that check. It should create a player metadata on creation!

Reply:
Great!